app/views/static/release_notes.html.erb

```diff
@@ -1,6 +1,16 @@
 <div class="container">
   <h1><strong>Poznámky k zmenám funkcionality a štruktúry dát</strong></h1>
   <div class="long-text">
+    <h2 class="anchorable">Zmena k 13. 8. 2019</h2>
+    <h3>Otvorené API</h3>
```

Comment:
Otvorene API alebo data? + API sa nemenilo?

Reply:
Pod co patri `upvs` schema? Na datahub API repe sa nic nemenilo priamo.

Reply:
- ak ide o zmenu na urovni DB, tak to parti pod "Otvorené dáta"
- davame cez datahub UPVS von aj ako API? ak ano, treba to tu doplnit aj pod "Otvorené API"